src/components/ble/NimbleController.cpp

```diff
@@ -105,17 +126,11 @@ void NimbleController::Init() {
 }
 
 void NimbleController::StartAdvertising() {
-  int rc;
-
-  /* set adv parameters */
   struct ble_gap_adv_params adv_params;
   struct ble_hs_adv_fields fields;
-  /* advertising payload is split into advertising data and advertising
-     response, because all data cannot fit into single packet; name of device
-     is sent as response to scan request */
   struct ble_hs_adv_fields rsp_fields;
+  int rc;
```

Comment:
I know you didn't add this, but what does this variable do? Seems very nondescript.

Also maybe an fixed-size integer would be better here?

Reply:
I relocated the variable to where it is used to make the purpose clearer. Note that this variable will be optimized out by the compiler unless debugging is enabled.

This code is essentially extracted from nimble and I'm following their convention of naming the return code "rc"  (I prefer rv for "return value"). The called nimble functions all return a full int.